tests/features/content-default-language.feature

```diff
@@ -0,0 +1,17 @@
+@api
+Feature: Content initial language.
+  In order to be create content
+  As a editor
+  I want to make sure that it impossible for a content editor to create an initial version of a content in any language other
```

Comment:
I want to make sure that it is not possible for a content editor to create a node's initial version in any language (line break here)
other than the site's default language